graphspace/settings/base.py

```diff
@@ -10,9 +10,12 @@
 
 from sqlalchemy.ext.declarative import declarative_base
 
+from dotenv import load_dotenv
```

Comment:
Also, can you add default .env file to PR? After that commit, you should add .env to [.gitignore](https://github.com/Murali-group/GraphSpace/blob/master/.gitignore). This way we wouldn't commit any changes to this file by mistake (similar to production.py file). 

Would we get an error if .env file is missing?

Reply:
I added the .env file to the branch with default variable values and added it to .gitignore
From testing, no .env file is needed to run Graphspace since base.py defaults the environment values if they are missing.